File: cerebral-cartography/cerebral-cartography/Form1.cs
namespace cerebral_cartography
{
    public partial class Form1 : Form
    {
        private bool shouldDrawCircle = false;

        public Form1()
        {
            InitializeComponent();
        }

        private void btnAddCircle_Click(object sender, EventArgs e)
        {
            var circle = new CircleControl
            {
                Size = new Size(100, 100),
                Location = new Point(10, 10)
            };
            this.Controls.Add(circle);
        }
    }
}

File: Form1.vb
Public Class Form1
    Private Sub btnAddTitle_Click(sender As Object, e As EventArgs) Handles btnAddTitle.Click
        Dim newOval As New OvalControl()
        newOval.Text = "Sample text that wraps inside the oval"
        newOval.Font = New Font("Arial", 10)
        newOval.Location = New Point(50, 50)
        Me.Controls.Add(newOval)
    End Sub
End Class
